src/content/blog/why-speak-at-jsheroes.md

```diff
@@ -0,0 +1,39 @@
+---
+author: alex-moldovan
+title: Why speak at JSHeroes
+published: Nov 8, 2024
+tags: ["cfp"]
+summary: A few reasons to convince you to apply as a speaker
+---
+
+In 2025 we’ll be running JSHeroes for the 7th time. In all our past editions we had amazing speakers on our stage. Maybe some of you reading this are already convinced and you want to be a speaker at our event, but for everyone else, let me try to articulate the "why".
+
+## Recognition, exposure and benefits
+
+Our conference has grown steadily and has a very good reputation among tech events. As we have a single track with generous breaks, you will be part of a rather small group of speakers that take the stage every year. Your talk will be seen live by 500+ attendees and then it will be available on our youtube channel to be shared with the entire world.
+
+But we know recognition and exposure don't pay the bills, so you will find that we offer one of the most complete speaker packages out there.
+
+First of all, we pay for your flights and book for you up to 4 nights at the 5 star hotel where the event takes place. We also redistribute the small profit at the end of the event (usually around 10K EUR - link to transparency) in the form of speaker fees. We’re very proud to have been able to pay 500 EUR to our speakers for the past two editions.
+
+On top of that, we arrange for airport transfers to and from the hotel, we offer SIM cards upon arrival and we invite you to a variety of activities with the community and with the organizing team throughout the week.
+
+## The speaker experience
+
+If there’s one thing we “obsess” over, it is the speaker experience. We understood from the early days that speakers coming in from all over the world are the best advocates for our event in the future.
+
+The speaker experience goes beyond the benefits. It encompasses things like having a great MC to prepare the audience for your talk, or making sure your talk is scheduled at the right time of the day so you can bring your best self up on stage. We offer help and support throughout the months before the event, in case you need to rehearse your talk or just brainstorm new ideas.
```

Comment:
```suggestion
We understood from the early days that speakers coming in from all over the world are the best advocates for our event in the future, so the speaker experience is something we are obsessed with improving every year. 

It goes beyond the benefits. It encompasses things like having a great MC to prepare the audience for your talk, or making sure your talk is scheduled at the right time of the day so you can bring your best self up on stage.

We offer help and support throughout the months before the event, in case you need to rehearse your talk or just brainstorm new ideas.
```